packages/flame/lib/src/camera/viewfinder.dart

```diff
@@ -83,7 +78,7 @@ class Viewfinder extends Component
   }
 
   /// Reference to the parent camera.
-  CameraComponent get camera => parent;
+  CameraComponent get camera => parent! as CameraComponent;
```

Comment:
why is this better?

Reply:
It's not, but we have to be able to check if `parent` is null at some places and `parentIsA` doesn't allow that.